response.json

```diff
@@ -0,0 +1 @@
+{"total":0,"p":1,"ps":100,"paging":{"pageIndex":1,"pageSize":100,"total":0},"effortTotal":0,"issues":[],"components":[],"facets":[]}
```

Comment:
what is it? 🤔

Reply:
I'm sorry. I remember I was running sonar locally. I think ```.gitignore``` doesn't cover it. Removed.